src/main/java/com/cdancy/bitbucket/rest/features/InsightsApi.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.features;
+
+import com.cdancy.bitbucket.rest.annotations.Documentation;
+import com.cdancy.bitbucket.rest.domain.common.RequestStatus;
+import com.cdancy.bitbucket.rest.domain.insights.Annotation;
+import com.cdancy.bitbucket.rest.domain.insights.AnnotationsResponse;
+import com.cdancy.bitbucket.rest.domain.insights.InsightReport;
+import com.cdancy.bitbucket.rest.domain.insights.InsightReportPage;
+import com.cdancy.bitbucket.rest.fallbacks.BitbucketFallbacks;
+import com.cdancy.bitbucket.rest.filters.BitbucketAuthenticationFilter;
+import com.cdancy.bitbucket.rest.options.CreateAnnotations;
+import com.cdancy.bitbucket.rest.options.CreateInsightReport;
+import com.cdancy.bitbucket.rest.parsers.RequestStatusParser;
+import org.jclouds.javax.annotation.Nullable;
+import org.jclouds.rest.annotations.BinderParam;
+import org.jclouds.rest.annotations.Fallback;
+import org.jclouds.rest.annotations.RequestFilters;
+import org.jclouds.rest.annotations.ResponseParser;
+import org.jclouds.rest.binders.BindToJsonPayload;
+
+import javax.inject.Named;
+import javax.ws.rs.Consumes;
```

Comment:
:warning: Wrong lexicographical order for 'javax.ws.rs.Consumes' import. Should be before 'org.jclouds.rest.binders.BindToJsonPayload'. (#QR30:39#)-CHECKSTYLE_CustomImportOrderCheck